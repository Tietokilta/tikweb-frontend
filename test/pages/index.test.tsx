/**
 * @jest-environment jsdom
 */

describe("Dummy", () => {
  test("Dummy", () => {
    expect(true).toBeTruthy()
  })
})
