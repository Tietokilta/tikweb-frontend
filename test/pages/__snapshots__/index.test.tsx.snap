// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/index.tsx Renders correctly 1`] = `
<div
  class="flex flex-col items-center flex-grow min-h-[100vh] bg-white"
  style="align-items: stretch;"
>
  <div
    class="z-50 sticky top-0 left-0 flex justify-center w-full h-12 px-4 md:px-8 lg:px-16 bg-black text-white"
  >
    <div
      class="w-full flex justify-between max-w-[90rem]"
    >
      <a
        class="sticky flex items-center font-normal text-2xl text-white"
        href="/"
      >
        <img
          alt="logo"
          class="w-8 h-8 mr-2"
          src="test-file-stub"
        />
        Tietokilta Ry
      </a>
      <nav
        class="hidden md:flex items-center font-mono text-lg"
      >
        <a
          class="font-mono ml-6 lg:ml-12 text-white"
          href="#"
        >
          Tapahtumat
        </a>
        <a
          class="font-mono ml-6 lg:ml-12 text-white"
          href="#"
        >
          Kilta
        </a>
        <a
          class="font-mono ml-6 lg:ml-12 text-white"
          href="#"
        >
          Fuksit
        </a>
        <a
          class="font-mono ml-6 lg:ml-12 text-white"
          href="#"
        >
          Abeille
        </a>
        <a
          class="font-mono ml-6 lg:ml-12 text-white"
          href="#"
        >
          Yritykset
        </a>
      </nav>
      <div
        class="flex md:hidden items-center"
      >
        <button
          class="text-white w-8 h-8 relative"
          type="button"
        >
          <span
            class="sr-only"
          >
            Open main menu
          </span>
          <div
            class="block w-5 absolute left-1/2 top-1/2 transform -translate-x-1/2 -translate-y-1/2"
          >
            <span
              aria-hidden="true"
              class="block absolute h-0.5 w-5 bg-current transform transition duration-500 ease-in-out -translate-y-1.5"
            />
            <span
              aria-hidden="true"
              class="block absolute h-0.5 w-5 bg-current transform transition duration-500 ease-in-out "
            />
            <span
              aria-hidden="true"
              class="block absolute h-0.5 w-5 bg-current transform transition duration-500 ease-in-out translate-y-1.5"
            />
          </div>
        </button>
        <div
          class="fixed top-0 left-0 h-full bg-gray-darkest transition-[max-width] ease-in-out duration-500 max-w-0 w-0 overflow-hidden"
        >
          <div
            class="p-4"
          >
            <a
              class="block font-mono text-2xl p-4 text-white"
              href="#"
            >
              Tapahtumat
            </a>
            <a
              class="block font-mono text-2xl p-4 text-white"
              href="#"
            >
              Kilta
            </a>
            <a
              class="block font-mono text-2xl p-4 text-white"
              href="#"
            >
              Fuksit
            </a>
            <a
              class="block font-mono text-2xl p-4 text-white"
              href="#"
            >
              Abeille
            </a>
            <a
              class="block font-mono text-2xl p-4 text-white"
              href="#"
            >
              Yritykset
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="w-full flex-grow"
  >
    <div
      class="w-full bg-black flex justify-center"
    >
      <div
        class="w-full relative h-[calc(100vh-48px-48px)]"
      >
        <div
          class="w-full h-full bg-cover bg-no-repeat bg-center"
          style="background-image: url(test-file-stub);"
        />
        <div
          class="absolute top-0 left-0 h-full w-1/2 from-black bg-gradient-to-r font-mono flex flex-col justify-end"
        >
          <div
            class="md:text-5xl sm:text-4xl p-16 text-white leading-tight"
          >
            <p
              class="mb-8"
            >
              Kiinnostaako tietotekniikan opiskelu?
            </p>
            <a
              class="lg:text-4xl md:text-3xl sm:text-2xl whitespace-nowrap inline-block bg-orange px-8 py-4 font-sans text-white uppercase tracking-wider"
              href="/"
            >
              Lue lisää
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      class="justify-center flex"
    >
      <div
        class="w-full relative p-3"
      >
        <h4
          class="font-mono text-xl"
        >
          Tietokilta lyhyesti
        </h4>
        <p
          class="font-sans pb-3 text-sm"
        >
          Tietokilta ry (TiK) on Aalto-yliopiston Perustieteiden korkeakoulun
           
          <a
            href="/"
          >
            tietotekniikan koulutusohjelman
          </a>
           opiskelijoiden ainejärjestö eli kilta, joka on avoin muillekin killan toiminnasta kiinnostuneille.
        </p>
        <p
          class="font-sans pb-3 text-sm"
        >
          Tietokilta järjestää opiskelijoilleen erilaista vapaa-ajan toimintaa niin Otaniemessä kuin sen ulkopuolellakin, valvoo heidän etujaan korkeakoulun toimielimissä, huolehtii yhteyksistä muihin opiskelijajärjestöihin ja pyrkii muuten huolehtimaan heidän hyvinvoinnistaan. Kaikki killan toiminta perustuu hyvien teekkariperinteiden mukaiseen vapaaehtoistyöhön ja yhteiseen talkoohenkeen.
        </p>
        <p
          class="font-sans pb-3 text-sm"
        >
          Kampuksella tietokiltalaiset löytää parhaiten kiltahuoneelta eli kiltikseltä, joka sijaitsee Tietotekniikan Talolla osoitteessa
           
          <a
            href="/"
          >
            Konemiehentie 2
          </a>
           rakennuksen koilliskulmassa.
        </p>
        <p
          class="font-sans pb-3 text-sm"
        >
          TiK toimii kaikkien Aalto-yliopiston opiskelijoiden etuja ajavan
           
          <a
            href="/"
          >
            Aalto-yliopiston ylioppilaskunnan
          </a>
           (AYY) piirissä.
        </p>
      </div>
      <div
        class="w-full p-3"
      >
        <h4
          class="font-mono text-xl"
        >
          Juuri Nyt
        </h4>
        <div
          class="font-mono text-xl rounded-xl shadow-md bg-gray-darkest text-white p-4"
        >
          <div
            class="flex flex-row justify-between"
          >
            <div
              class="flex-column"
            >
              <p
                class="font-mono"
              >
                Wappusitsit
                 @ 
                Smökki
              </p>
              <p
                class="font-mono text-sm"
              >
                su 31-07-2022, klo 19:57
              </p>
            </div>
            <div
              class="flex-column justify-end py-1 px-3 rounded-2xl my-auto h-fit flex-shrink bg-white text-black font-mono text-sm font-bold"
            >
              76
              /
              250
            </div>
          </div>
          <div
            class="relative flex-row mt-3 text-sm font-sans"
          >
            Teekkareiden vuoden tärkein tapahtuma lähenee ja mikä olisi parempi tapa juhlistaa sitä jo ennakkoon kuin Wappusitsit! Tänäkin vuonna paikan päällä raikaa
            <div
              class="absolute top-0 left-[-5px] h-[calc(100%+10px)] w-[calc(100%+10px)]"
            />
          </div>
          <div
            class="relative flex-row mt-3 font-mono text text-sm"
          >
            <a
              href="https://tietokilta.fi"
            >
              Ilmoittaudu
            </a>
          </div>
        </div>
        <h4
          class="font-mono text-xl pt-3"
        >
          Tulevat tapahtumat
        </h4>
        <div
          class="mb-3 font-mono text-xl rounded-xl shadow-md bg-gray-darkest text-white p-4"
        >
          <div
            class="flex flex-row justify-between"
          >
            <div
              class="flex-column"
            >
              <p
                class="font-mono"
              >
                Nappusitsit
                 @ 
                Smökki
              </p>
              <p
                class="font-mono text-sm"
              >
                su 31-07-2022, klo 19:57
              </p>
            </div>
            <div
              class="flex-column justify-end my-auto font-mono text-sm font-bold"
            >
              <a
                href="https://tietokilta.fi"
              >
                Lue lisää
              </a>
            </div>
          </div>
        </div>
        <div
          class="mb-3 font-mono text-xl rounded-xl shadow-md bg-gray-darkest text-white p-4"
        >
          <div
            class="flex flex-row justify-between"
          >
            <div
              class="flex-column"
            >
              <p
                class="font-mono"
              >
                Fappusitsit
                 @ 
                Smökki
              </p>
              <p
                class="font-mono text-sm"
              >
                su 31-07-2022, klo 19:57
              </p>
            </div>
            <div
              class="flex-column justify-end my-auto font-mono text-sm font-bold"
            >
              <a
                href="https://tietokilta.fi"
              >
                Lue lisää
              </a>
            </div>
          </div>
        </div>
        <div
          class="mb-3 font-mono text-xl rounded-xl shadow-md bg-gray-darkest text-white p-4"
        >
          <div
            class="flex flex-row justify-between"
          >
            <div
              class="flex-column"
            >
              <p
                class="font-mono"
              >
                Sappusitsit
                 @ 
                Smökki
              </p>
              <p
                class="font-mono text-sm"
              >
                su 31-07-2022, klo 19:57
              </p>
            </div>
            <div
              class="flex-column justify-end my-auto font-mono text-sm font-bold"
            >
              <a
                href="https://tietokilta.fi"
              >
                Lue lisää
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="flex w-full text-white justify-center bg-gray-darkest px-4 md:px-8 lg:px-16 py-6"
  >
    <div
      class="w-full max-w-[90rem]"
    >
      Footer
    </div>
  </div>
</div>
`;
